src/main/java/Storage.java

```diff
@@ -0,0 +1,98 @@
+import java.io.File;
+import java.io.FileNotFoundException;
+import java.io.FileWriter;
+import java.io.IOException;
+import java.time.LocalDate;
+import java.time.format.DateTimeParseException;
+import java.util.ArrayList;
+import java.util.Scanner;
```

Comment:
Perhaps the imports from different packages could be separated to make it easier to browse the list?